crates/shared/Cargo.toml

```diff
@@ -31,7 +31,6 @@ hex = { workspace = true }
 hex-literal = { workspace = true }
 itertools = { workspace = true }
 lazy_static = { workspace = true }
-lru = "0.8"
```

Comment:
Replaced with the Cached crate that we already depend on and provides same functionality.